kotlin/kotest/src/commonTest/kotlin/karat/kotest/Counter.kt

```diff
@@ -9,6 +9,7 @@ import io.kotest.property.Arb
 import io.kotest.property.arbitrary.enum
 import karat.concrete.progression.Info
 import karat.concrete.progression.Step
+import karat.kotlin.test.should
```

Comment:
Is this needed now?

Reply:
Yes, because I've moved some of the shared code into a new module